### PR TITLE
detr-resnet50: scores->logits

### PR DESCRIPTION
**Pull request by Wovchena**

### CODE REVIEW

File: models/public/detr-resnet50/README.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ Expected color order is `BGR`.
     - `w` - width of bounding box(values are in normalized format, in range [0, 1])
     - `h` - height of bounding box(values are in normalized format, in range [0, 1])
 
-2. Scores, name: `scores`, shape - `1, 100, 92`. Contains scores for 91 [Common Objects in Context (COCO)](https://cocodataset.org/#home) object classes. The last class is `no-object` class.
+2. Scores, name: `scores`, shape - `1, 100, 92`. Contains logits for 91 [Common Objects in Context (COCO)](https://cocodataset.org/#home) object classes. The last class is `no-object` class.
```
Comment (eaidova) [inline models/public/detr-resnet50/README.md:69]: logints -> scores in logits format

Comment (eaidova) [reply models/public/detr-resnet50/README.md:69]:
```suggestion
2. Scores, name: `scores`, shape - `1, 100, 92`. Contains scores in logits format for 91 [Common Objects in Context (COCO)](https://cocodataset.org/#home) object classes. The last class is `no-object` class.
```

**Review by eaidova — APPROVED** [review]
